### ISSUE TITLE
График XO

### ISSUE THREAD

**Issue by verbadim** [SBProX/SBProX-Dev#1889]
Здравствуйте, добавьте пожалуйста график Крестики Нолики, было бы очень удобно

**Comment by qlewer** [comment]
Согласен, было бы удобно увидеть так же и другие популярные варианты, Ренко, Range-бары и т.п.
